### PR TITLE
Adding create component from git UI test

### PR DESCRIPTION
**Pull request by lgrossma**
see: https://github.com/redhat-developer/vscode-openshift-tools/issues/3068

### CODE REVIEW

File: .github/workflows/continuous-integration-workflow.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, windows-latest, ubuntu-latest]
-        node: [ '18' ]
+        node: [ '18.15.x' ]
```
Comment (vrubezhny) [inline .github/workflows/continuous-integration-workflow.yml:26]: 18.15.x  doesn't look like the is to install a latest version of Node.js 18 - do we have any limitations on Node.js version related to VSCode OS Tools and/or test?

**Review by vrubezhny — COMMENTED** [review]
Please see a question inline

**Comment by lgrossma** [comment]
@vrubezhny Yes, the VSCode extension tester used for UI tests does not work with higher node versions due to selenium incompatibility.

**Comment by vrubezhny** [comment]
> @vrubezhny Yes, the VSCode extension tester used for UI tests does not work with higher node versions due to selenium incompatibility.

@lgrossma Thanks! This probably was a reason why I couldn't update vscode ext. tester to 5.10.0 neither to 5.6.0.

**Review by datho7561 — APPROVED** [review]
Looks good, and seems to be passing consistently on my computer

**Pull request merged**